backend/file-system/src/main/java/com/htec/filesystem/service/FileService.java

```diff
@@ -181,6 +178,42 @@ public void saveFileIntoDB(String filePath, String fileName, long shelfId, long
         fileRepository.save(fileEntity);
     }
 
+    public void copyFile(Long fileId, Long shelfId, Long userId) {
+
+        List<ShelfEntity> shelfEntities = shelfRepository.findAllByUserId(userId);
+
+        FileEntity fileEntity = fileRepository.findById(fileId)
+                .orElseThrow(ExceptionSupplier.noFileWithGivenId);
+
+        shelfRepository.findById(shelfId)
```

Comment:
result of this is ignored? is that intentionally?

Reply:
Yes that is intentional since i just want to see if there is a shelf with given id